### ISSUE TITLE
RLS: 1.0.2

### ISSUE THREAD

**Issue by jbrockmendel** [pandas-dev/pandas#32415]
@pandas-dev/pandas-core think end of this week?

**Comment by TomAugspurger** [comment]
Friday sounds good to me, but I don't have a good sense for where things are at.

Any regressions that we definitely want to see fixed?

Any open PRs we'd like to see merged?

**Comment by jbrockmendel** [comment]
> Any regressions that we definitely want to see fixed?

- [ ] #32395 ill make a PR tomorrow morning
- [ ] #31925 has a viable-looking PR in #32416.
- [ ] #31827 I think we just need to say "yes"
- [ ] #32146 has a viable PR in #32242 
- [ ] #31774 mostly just needs a better exception message

Probably no action on
- [ ] #31978 looks like we havent been able to reproduce
- [ ] #31710 supposedly will be fixed by cython 0.30; would be a pain to fix on our end

Most of the rest of the regression issues from the last few weeks are groupby/agg-related, not obvious what it will take to fix.

**Comment by jreback** [comment]
> #31827 I think we just need to say "yes"

this was long deprecated and it is not clear from the OP what is going on - certainly not 1.02 material

**Comment by TomAugspurger** [comment]
I'm roughly targeting Wednesday for a release.

**Comment by charlesdong1991** [comment]
@TomAugspurger sorry to interrupt, maybe #31524 could also be nice to be included in 1.0.2? I've seen several people raise this issue. 

If someone could take a look at PR, I would try my best to give updates on the PR ASAP.

**Comment by TomAugspurger** [comment]
I plan to tag 1.0.2 after https://github.com/pandas-dev/pandas/pull/32658 is merged and backported.

**Comment by TomAugspurger** [comment]
OK, starting the release.

**Comment by TomAugspurger** [comment]
FYI, the last thing before we can upload to PyPI is the Windows wheels. @cgohlke do you know if you're able to build them for 1.0.2?

When https://github.com/MacPython/pandas-wheels/pull/74 is merged, windows wheels will be built as well. I wouldn't be surprised if we get issue reports from switching to those, but it'd be nice to have control of this stage too, rather than relying on Christoph's (extremely valuable) generosity.

**Comment by TomAugspurger** [comment]
OK, all the wheels are built. I'm planning to upload to the wheels today. I think having a release with the fixed regressions is valuable, despite the regression in https://github.com/pandas-dev/pandas/issues/32685. And conda-forge is already out there.

**Comment by jorisvandenbossche** [comment]
@TomAugspurger Thanks a lot for doing the release!

Agreed we can do a 1.0.3 afterwards to include the new regression.

**Comment by jreback** [comment]
yep thanks @TomAugspurger awesome as always

**Comment by TomAugspurger** [comment]
Released on PyPI now.
